h2o-persist-hdfs/src/main/java/water/persist/VecFileSystem.java

```diff
@@ -35,7 +35,10 @@ public void initialize(URI name, Configuration conf) throws IOException {
   }
 
   @Override public FileStatus getFileStatus(Path p){
-    return new FileStatus(_v.length(),false,1,_v.length()/_v.nChunks(),0l,VecFileSystem.VEC_PATH);
+    if (VEC_PATH.equals(p)) {
+      return new FileStatus(_v.length(),false,1,_v.length()/_v.nChunks(),0l, VecFileSystem.VEC_PATH);
+    } else
+      return null;
```

Comment:
Why return null ? In open() method, IllegalArgumentExceptino is thrown. In listStatus method,an empty array of FileStatus is returned.

Reply:
You are right, it should throw FileNotFoundException. I need to check what Parquet does and why I put it there this way.